test/test_jit_py3.py

```diff
@@ -458,6 +458,17 @@ def annotated_fn(x: torch.Tensor) -> Tuple:
         with self.assertRaisesRegex(RuntimeError, r"Attempted to use Tuple without a contained type"):
             torch.jit.script(annotated_fn)
 
+    def test_tuple_subscripted_assign(self):
```

Comment:
this doesn't need to go in `test_jit_py3`, it can just go in `test_jit` now that we always run py3. Actually we should get rid of this file entirely :P

Reply:
Hmm, don't we still need smaller `test_jit_py2` to validate that comment style type annotation are still recognized by JIT?